### ISSUE TITLE
Typehead on-blur

### ISSUE THREAD

**Issue by alexivenkov** [wffranco/vue-strap#55]
Hello!
How can i handle blur event in typeahead plugin?  For example when I type something, dont select suggestion values and switch focus to another form component, i still want to save the value into vuex store.

**Comment by etlweather** [comment]
I don't know about vuex store because I haven't learned how that works yet, but you can set a `v-model` property on the typeahead which will have the value typed in the field even when not selected from the suggestions.

**Comment by etlweather** [comment]
But I would like to be able to use the blur event myself for other reasons.

**Comment by TonyRyan** [comment]
Did you find a way to do this?

**Comment by alexivenkov** [comment]
Yes, thanks to @etlweather answer. I set a `v-model`  property and link it with computed set/get [(doc)](https://vuejs.org/v2/guide/computed.html#Computed-Setter) method.
